### ISSUE TITLE
Width of CJK character change when line contains escaped chars

### ISSUE THREAD

**Issue by QiangF** [ivanceras/svgbob#38]
Normally one CJK character equal two english character width, the following will be closed.

    +----+
    | 一 |
    +----+

If the line contains escape qutoe, the CJK character seems need 3 english character spaces to get
the shape closed.

    +--------+
    |"""一"""|
    +--------+

I am using svgbob 0.3.2.

**Comment by QiangF** [comment]
If there is no extra spaces between CJK characters, the quote is not needed. Is it possible to prevent each CJK characters being treated as a separate element as described in the first FAQ?

**Comment by m-h-k** [comment]
I have a similar issue with German "umlauts" and svgbob 0.4.1:

```
+----------+
| ÖÄÜ      |
+----------+

+----------+
| "ÖÄÜ"    |
+----------+

+----------+
| "ÖÄÜ" |
+----------+
```

will be converted to

![test](https://user-images.githubusercontent.com/46789488/106154649-1cb0f780-6180-11eb-884b-255a42eab622.png)

**Comment by ivanceras** [comment]
This even panics in the new version of svgbob (0.5.3) . I'm reviewing old code what has make this code panics.

**Comment by ivanceras** [comment]
Fixed in this commit 5a2eae141b9c7139ff59fe8e7823cb531cc7d363

**Comment by ivanceras** [comment]
Published svgbob `0.5.5`, with a fix on this. 
Online [editor](https://ivanceras.github.io/svgbob-editor/) works as well.
